polaris-migrator/src/migrate.ts

```diff
@@ -53,6 +53,7 @@ export async function migrate(
       runInBand: true,
       silent: false,
       stdin: false,
+      ...options,
```

Comment:
It looks like we can remove the above `dry` and `print` options with this addition